tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -32,6 +35,24 @@ const DayComponent = ({ date, state, marking, navigation }) => {
 }
 
 export const Calendar = ({navigation}) => {
+    const [calendarData, setCalendarData] = useState({});
+
+    //updates the data when you come back to the calendar screen
+    useEffect(() => {
+        const unsubscribe = navigation.addListener('focus', () => {
+            getCalendarData();
+        });
+        return unsubscribe;
+    }, [navigation]);
```

Comment:
When does `navigation` change such that it triggers the useEffect? I'm not super familiar with how the navigation library works

Reply:
@edwardhuahan I think you should take a look at this part to make sure it works with what you did with the filters

Reply:
It should trigger when it is used. At least that's what I thought.